cobspec/src/main/java/com/saralein/cobspec/logger/ApplicationLogger.java

```diff
@@ -0,0 +1,44 @@
+package com.saralein.cobspec.logger;
+
+import com.saralein.cobspec.data.LogStore;
+import com.saralein.server.logger.Logger;
+import java.io.PrintStream;
+import java.time.Instant;
+
+public class ApplicationLogger implements Logger {
+    private PrintStream printer;
+    private LogStore logStore;
+
+    public ApplicationLogger(PrintStream printer, LogStore logStore) {
```

Comment:
We don't need to do this now necessarily, but it seems like we would want the `ApplicationLogger` to collaborate with a single type, instead of `PrintStream` and `LogStore`.

Reply:
This is also a violation of ISP

Reply:
OK, will address this on perhaps a different PR in the future.